catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -2482,73 +2418,45 @@ protected ResourceResponse getResource(ResourceRequest resourceRequest, boolean
                 resourceSourceName = resolvedSourceId;
             }
 
-            String key;
-            try {
-                key = new CacheKey(metacard, resourceRequest).generateKey();
```

Comment:
I saw this when I reviewed the diff before. I'm starting to think that `CacheKey` should be `CacheKeyGenerator` instead. Thoughts?